homeassistant/components/asuswrt/config_flow.py

```diff
@@ -201,6 +203,12 @@ async def async_step_init(self, user_input=None):
                         CONF_CONSIDER_HOME, DEFAULT_CONSIDER_HOME.total_seconds()
                     ),
                 ): vol.All(vol.Coerce(int), vol.Clamp(min=0, max=900)),
+                vol.Optional(
+                    CONF_TRACK_NEW,
```

Comment:
We shouldn't add this option. We already have the system option.

Reply:
The reason for adding the option is to avoid devices creation. I know that we already discuss about this. but system option create devices in status disabled and sometimes there are devices that we don't want to have (may be because related to a guest or generated by device that use random mac-address). Once new devices are created, the only way to remove them is to completely remove the integration.

Reply:
I don't think we should add this per integration. If we need this feature it should be application wide. Please open an architecture issue in that case.

Reply:
Understandable, how should I open architecture issue? What is the right path?

Reply:
Start by explaining the problem that you see. Then if you have solutions, you explain them as well.

https://github.com/home-assistant/architecture/discussions

Reply:
This PR also contain some fix for sensor tests, but probably is better to open a new one, right?

Reply:
Yes. Please always separate fixes from feature additions and do one thing per PR.

Reply:
>  Once new devices are created, the only way to remove them is to completely remove the integration.

User practically is forced to edit the hidden .storage files now, because that way the devices and entities can also be removed. But, of course that is not advised, and hardly a user friendly solution. Simply disabling the multitude of added devices isn't either. If only because of the names they take in the system.